pkg/unity/find-line-mappings.go

```diff
@@ -48,3 +48,42 @@ func GetAndroidLineMapping(path string, buildDir string) (string, error) {
 
 	return "", fmt.Errorf("Unable to fine line mapping file in your project: %s ", buildDir)
 }
+
+// GetiOSLineMapping locates the LineNumberMappings.json file for iOS builds.
+//
+// This function attempts to resolve the path to the IL2CPP line number mapping file,
+// used for symbolication or debugging in iOS builds. The resolution follows this order:
+//
+//  1. If the input 'path' is non-empty, it is returned as-is.
+//  2. It checks the default path: Library/Bee/artifacts/iOS/il2cppOutput/cpp/Symbols/LineNumberMappings.json.
+//  3. If not found, it searches under a backup folder (ending with "_xcode" inside 'projectRoot') at:
+//     Il2CppOutputProject/Source/il2cppOutput/Symbols/LineNumberMappings.json.
+//
+// Parameters:
+//
+//	path        - an optional explicit path to the mapping file. If provided, it is returned directly.
+//	projectRoot - the root path of the Unity project, used to search for backup artifacts.
+//
+// Returns:
+//
+//	mappingPath - the resolved path to LineNumberMappings.json.
+//	error       - non-nil if the file cannot be found or the backup folder is missing.
+func GetiOSLineMapping(path string, projectRoot string) (string, error) {
```

Comment:
I'm not sure this is the correct casing – `GetIOSLineMapping`?

Reply:
You are correct, everywhere else we have it as `Ios` if its in the middle of a function name